targets/csharp/source/source/Uunit/tests/PlayFabApiTest.cs

```diff
@@ -537,10 +480,169 @@ private void GetObjects2Continued(PlayFabResult<GetObjectsResponse> result, UUni
             if (!int.TryParse(result.Result.Objects[TEST_DATA_KEY].EscapedDataObject, out int actualValue))
                 actualValue = -1000;
             testContext.IntEquals(_testInteger, actualValue, "Failed: " + _testInteger + "!=" + actualValue + ", Returned json: " + result.Result.Objects[TEST_DATA_KEY].EscapedDataObject);
-
             testContext.EndTest(UUnitFinishState.PASSED, null);
         }
 #endif
+        /// <summary>
+        /// CLIENT API
+        /// Test that Multiple login can be done
+        /// </summary>
+        [UUnitTest]
+        public void MultipleLogin(UUnitTestContext testContext)
```

Comment:
Please change the name to indicate that it's a multiple login using the static interface.
The next team is going to come behind you and test multiple login using the instance interface, and they keep having to rename these tests.

Reply:
The other test (for instanced APIs) is in another test class, but i agree it's better to rename this method as you proposed

---
In reply to: [257055384](https://github.com/PlayFab/SDKGenerator/pull/404#discussion_r257055384) [](ancestors = 257055384)